13/go-db/code/batchInsert.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"database/sql"
+	"fmt"
+	"strconv"
+
+	_ "github.com/go-sql-driver/mysql"
+)
+
+func main() {
+	db, err := sql.Open("mysql", "root:123@tcp(localhost:3306)/tmp")
```

Comment:
Man, you committed database password to GitHub :trollface:

Reply:
just local stuffs :P

Reply:
@huytd we should remove that password and reset everything to the first commit, right? :trollface:

Reply:
Squash everything in the history into just 1 commit :trollface: